_states/ext_state_qvm.py

```diff
@@ -76,11 +100,11 @@ def __virtual__():
     '''
     # XXX: TEMP
     if not hasattr(qubes_utils, '__opts__'):
-        qubes_utils.__opts__ = __opts__
+        qubes_utils.__opts__ = __opts__  # NOQA
```

Comment:
Yes, something like this. As this is purely to make static analysis happy I think it could be just one block at the top of module (to not repeat it in each function), but it's ok this way too.

> @marmarek flake8 i think. But since i cleaned up my vim/synaptics-checker pipeline and only use pylint right now and we can kick it.

Whatever you like, if that would help you, I'm ok with `# NOQA` comments.